### DISCUSSION TITLE
Unable to upgrade organization from Free to Team plan

### DISCUSSION THREAD

**Discussion by gruisinger** [community/community#53609] [GitHub Education]
Similar to these two unanswered questions from last month:
https://github.com/orgs/community/discussions/50516
https://github.com/orgs/community/discussions/50702

I am unable to renew the free Team plan for [my organization](https://github.com/kiteaai). I did get the option:

_Upgrade your academic organizations
As a teacher, you have the ability to upgrade your academic organizations used for your classroom, student groups, or research to a GitHub Team plan at no cost._

I clicked the "Upgrade to GitHub Team" button, but it has been over two weeks with no change in our plan status. Are there any other steps to take? I have recently reverified the organization as well.

**Comment by samus-aran** [comment]
Hi @gruisinger 👋 

What error appears when you click on "Upgrade to GitHub Team"? We can see that this org has been upgraded so it would be great to identify what errors you are experiencing 👍

**Reply by gruisinger** [reply]
There was no error,  and the option to upgrade the organization no longer appears.

**Reply by samus-aran** [reply]
@gruisinger So that sounds like it has registered the org upgrade 👍  Is there evidence that you see that it hasn't upgraded properly?

**Reply by gruisinger** [reply]
because the organization is on the free plan still:

_Current plan_
 
_GitHub Free_
_The basics for organizations and developers_

and when I look at the compare plans option, I see:

_Team_
_Advanced collaboration & support_

_$4_
_per user/month_

We also no longer have branch protection rules, and this is the explanation:

_Your protected branch rules won't be enforced on this private repository until you upgrade your organization account to a GitHub Team or Enterprise account._

**Reply by emmnull** [reply]
@samus-aran, [our organisation](https://github.com/CUPUM) is facing the exact same situation. The prof. managing the organisation received an email a few weeks back warning us that a "coupon" we had applied for was soon reaching expiry. After submitting a request to reapprove his professoral status on GitHub and asking for a team plan upgrade, we did not get any error but the org's settings still display that we are running on GitHub Free. We tried emailing support but haven't received any answer. Is there a way to confirm what plan our org is currently benefiting from?

**Reply by marcoeilers** [reply]
I have the exact same problem with [my organization](https://github.com/viperproject).

**Reply by samus-aran** [reply]
Thank y'all for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by mihai-sysbio** [reply]
Same problem for my organization [MetabolicAtlas](https://github.com/MetabolicAtlas)

**Reply by micahgallen** [reply]
My organization has the exact problem: 
https://github.com/embodied-computation-group

I reverified weeks ago and also clicked the "upgrade organization" button on the Global University website, and nothing has happened. 

This feels a bit like a trap. Lure public research labs into using github organizations with the free offer, and then rug pull us once our labs are fully dependent upon it! Really bad form here - I hope this is quickly resolved.

**Reply by mihai-sysbio** [reply]
> Same problem for my organization [MetabolicAtlas](https://github.com/MetabolicAtlas)

Resolved.

**Comment by samus-aran** [comment]
Hi @gruisinger 👋 

We've had to revoke your existing Organisation upgrade coupon in order to progress with a solution. Please can you try and upgrade your organisation again and let us know if this has helped 👍 

@marcoeilers & @iolyd we are still confirming that this resolves the issue so we'll update you when we will apply the same method to your organisations. Thank you for your patience.

**Reply by gruisinger** [reply]
Thanks, I checked from https://education.github.com/globalcampus/teacher and once again had the option to upgrade my academic organization. I clicked the upgrade button, and saw a message at the top of the page that said "You have successfully submitted your organization upgrade request". I will let you know if this gives the ability to upgrade to the Team plan for free (so far it does not, but I clicked the upgrade button mere minutes ago).

**Reply by gruisinger** [reply]
I will also note, that the wording on the website says: _"As a teacher, you have the ability to upgrade your academic organizations used for your classroom, student groups, or research to a GitHub Team plan at no cost."_

However the wording in the email I received says: "We have upgraded </my organization/> to an organization plan with unlimited free private repositories and unlimited seats."

These are not the same thing- the Team plan offers other benefits, including branch protection rules on private repositories.

**Reply by gruisinger** [reply]
also note the response from yesterday on [this post](https://github.com/orgs/community/discussions/50516):

_GitHub support team finally came through and upgraded our team manually._

**Reply by samus-aran** [reply]
Hi @gruisinger 

>  I will let you know if this gives the ability to upgrade to the Team plan for free (so far it does not, but I clicked the upgrade button mere minutes ago).

Just to confirm, in your billing page of that organisation it still states its still on the free plan?

> I will also note, that the wording on the website says: "As a teacher, you have the ability to upgrade your academic organizations used for your classroom, student groups, or research to a GitHub Team plan at no cost."
> 
> However the wording in the email I received says: "We have upgraded </my organization/> to an organization plan with unlimited free private repositories and unlimited seats."

There are no limitations of features for teachers when upgrading organisations to teams. But I agree that the wording on the notification and email are different.

> also note the response from yesterday on [this post](https://github.com/orgs/community/discussions/50516):

> GitHub support team finally came through and upgraded our team manually.

Appreciate the call out. We would ideally want to resolve any underlying issues that seems to be stopping our teachers from upgrading their organisations by themselves and restore the self-serve aspect of the upgrade. Thank you for your participation and your contribution to help us help others.

**Reply by gruisinger** [reply]
hello @samus-aran 

> Just to confirm, in your billing page of that organisation it still states its still on the free plan?

that is correct. I just checked again, and nothing has changed, my orgnaiztion is still on the free plan, and the Team plan shows as having an upgrade cost. 

> Appreciate the call out. We would ideally want to resolve any underlying issues that seems to be stopping our teachers from upgrading their organizations by themselves and restore the self-serve aspect of the upgrade. Thank you for your participation and your contribution to help us help others.

My organization is grateful for the immense value GitHub provides to us as an educational institution, and I recognize it is important to support the community when we can. However this problem has been ongoing for over a month, and at this point I think it is fair to pass the torch to the next organization that encounters the issue.

**Reply by samus-aran** [reply]
Hi @gruisinger 

Appreciate your honesty and that is certainly a reasonable request. Are you able to provide a GitHub Support ticket number I can then escalate that would be much appreciated.

**Reply by gruisinger** [reply]
greetings @samus-aran 

I did indeed open a ticket a few days after the issue popped up, and it was closed and I was directed here. but if it helps: https://support.github.com/ticket/personal/0/2079440

**Reply by gruisinger** [reply]
Hello @samus-aran ,
Checking to see if there is anything else we can do to assist here. I checked the status of my organization account just now, and it remains unchanged. it is still free, and the option for Team is still a paid one. When I visit https://education.github.com/globalcampus/teacher I no longer have the option to upgrade my organization. I think we are at an impasse without further assistance. 
If there is some lead time to being able to take advantage of the feature described as "the ability to upgrade your academic organizations used for your classroom, student groups, or research to a GitHub Team plan at no cost" please let us know.

**Reply by jumarmartin** [reply]
Hi,

Wanted to +1 this as our organization is facing the same (@elonsoc) so our sponsor created a new organization (@elonesc) which places our organization in a less-than-desired position. Looking forward to also getting this resolved, if possible.

cc @Ryan-Mattfeld-CS

**Reply by emmnull** [reply]
@samus-aran Could we possibly get an update on this issue or a confirmed working procedure to fix things on our side? We are planning to explore the use of a few features only available with upgraded plans for the upcoming semester (private static pages, protected branches on private repos, etc.) but this is currently blocking us.

**Comment by gruisinger** [comment]
My organization has still not seen any change to our plan. we are still on the Free plan, and the Team plan shows a cost associated with it. We have had no updates since April 26 (about two weeks ago). I am not sure if @samus-aran is still working on it or not.

**Comment by rig221** [comment]
My organization is experiencing the same issue:

https://github.com/becky-work-check/

I have an approved education account through Github Education; I requested an upgrade for my organization to Team via the link on the Github Education dashboard; and my organization still remains on the free plan, asking me to pay in order to upgrade to the Team plan.  support@github.com is unresponsive.

Can someone on GitHub staff please take a look at this?

Additionally, I have a couple of more organizations I need to create and then upgrade to Team, and these are time-sensitive upgrades.  Am I expected to contact support (seems to be useless) or post here every time I need to create and upgrade an organization? I would think the process would be automatic for verified educator accounts.  This is very frustrating.

**Comment by ahnitz** [comment]
Our organization also appears to be hitting this issue also. I wonder if it has to do with the fact that we used to be on the old coupon before the new system? 

https://github.com/gwastro

To reiterate the behavior (which seems to be common here), our account says its been upgraded on the github education page, but if I go to our organization page it says we are on the free plan and there is no hint of a coupon as it shows a cost for upgrading to the teams plan. 

We'd appreciate any assistance in resolving this issue. Thank you!

**Reply by ahnitz** [reply]
I can add that if I go here https://education.github.com/discount_requests/application, I see the application (from back in February) was approved. Also also see that we have the same problem at 

github.com/sugwg

I was also the requester there and it is listed as approved, but there is no obvious discount to upgrade to teams and the plan is set to the free one.

**Comment by snuarc** [comment]
Our organization
https://github.com/SNU-ARC
is also having a similar issue.

I recently joined the GitHub Global Campus as a teacher, created a classroom, and also clicked the "Upgrade to GitHub Team" button in the Global Campus tab. I also got an e-mail that saying "We have upgraded @SNU-ARC to an organization plan with unlimited free private repositories and unlimited seats."

However, our organization https://github.com/SNU-ARC is still on the Free plan and I cannot find a way to upgrade it to GitHub Team without payment. I also made a request through support@github.com but did not get any response.
Can you manually upgrade our organization to GitHub Team?

**Comment by gruisinger** [comment]
I am updating this post since a few others have commented they are in similar predicaments since my previous comment on May 11, 2023. Nothing has changed for my organization. we are still on the free plan, and the Team plan still shows a cost associated with it if I go to the "billing/plans" page. We also attempted to address our issue via email, and had no luck there. 
At this point the only choice seems to be pay money to get the features that were previously free to academic organizations.  As of the time of this writing, those features still appear [here](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program):

> GitHub Campus Program offers GitHub Enterprise Cloud and GitHub Enterprise Server free-of-charge for schools that want to make the most of GitHub for their community.
> 
> GitHub Campus Program is a package of premium GitHub access for teaching-focused institutions that grant degrees, diplomas, or certificates. GitHub Campus Program includes:
> 
> No-cost access to GitHub Enterprise Cloud and GitHub Enterprise Server for all of your technical and academic departments
> 50,000 GitHub Actions minutes and 50 GB GitHub Packages storage
> Teacher training to master Git and GitHub
> Exclusive access to new features, GitHub Education-specific swag, and free developer tools from GitHub partners
> Automated access to premium GitHub Education features, like the GitHub Student Developer Pack

**Reply by emmnull** [reply]
At this stage, with how long this issue has been standing and the fact that it represents an unannounced regression in feature availability, it certainly is starting to feel like deceptive marketing.

**Comment by markhc** [comment]
Same issue with our organization. We even tried to actually upgrade to Team plan, but Github won't do it. So now we are stuck at free plan, with no way to do anything. Great job.

**Comment by nineil-weber** [comment]
Good evening,

I am a faculty at Weber State University, and I requested an upgrade from free to github team license, and I got a message like this "Congrats! Your request for wsu-fall23 was approved and your benefits will be available soon."

Do you know how long should I wait? or is a similar issue as other users that is not approved.

Thank you very much!

**Comment by tsikerdekis** [comment]
Hello,
It seems that I am having the same issue as everyone else. I click on upgrading my assignments organization but it has not been upgrade to teams for two days now. Not sure how long it takes typically but it would be good to have an estimate so that we can prepare for our classes.

**Reply by nineil-weber** [reply]
Hi @tsikerdekis . Mine took around 3-5 days, then you will received an email. Hope it helps!

**Comment by Langster2323** [comment]
I can't upgrade my project to a team plan, after clicking the button "Next, customize your setup"
I receive these errors:
'undefined' is not a valid input.
and
There were problems creating your organization.

**Comment by chektien** [comment]
I have a similar problem under https://github.com/sit-dsa . I clicked upgrade and it is not reflected yet. Thanks.

**Comment by pinventado** [comment]
Same problem here. Did anyone get this resolved? https://github.com/CSUF-CPSC121L-2023F

**Comment by cmatKhan** [comment]
Bump -- we are having the same problem. We hit the upgrade button ~5 days ago and there has been no upgrade. However, the organization no longer is shown in the dropdown of possible organization accounts which could be upgraded on the globalcaompus page.

https://github.com/CSE587A

**Reply by cmatKhan** [reply]
We ultimately did get upgraded, but it took weeks

**Comment by KrisJordan** [comment]
Also seeing this @samus-aran

I requested benefits with two orgs for courses that started on Monday.

One a week ago, receiving benefits applied ❤️ email now 3 or 4 days ago, still on Free.

The second three or four days ago, also received benefits applied email, still on Free.

It seems like my only solution is going to be to wipe and reuse an old semester's org, which is suboptimal on many fronts. One assignment already needed to be modified due to the lack of Teams, in my Software Engineering course.

Any help with this upgrade path is highly, highly appreciated. It seems something has broken down in the integration with billing.

**Comment by smasongarrison** [comment]
I also have the same issue

**Comment by sonils-digital** [comment]
https://github.com/sonils same issue

You cannot change your plan because this organization is currently being upgraded to an Enterprise Account. Please wait until the payment has finished processing.

**Comment by hhibshi** [comment]
I am having the same issue with my organization. I opened a ticket and no response yet.

**Comment by smasongarrison** [comment]
When did you open your ticket? I opened mine in mid september, and have
still not heard back other than to acknowledge that they received it

**Reply by hhibshi** [reply]
I opened it early this week.

**Reply by ahnitz** [reply]
I opened a ticket in June. No response yet. I think we can surmise that they aren't going to respond to tickets related to this issue.

**Reply by smasongarrison** [reply]
I've just repinged them on my support ticket.

**Reply by smasongarrison** [reply]
Still nothing. :/

**Comment by gruisinger** [comment]
I checked the GitHub education page today and once again had an "upgrade" button. I clicked it and it appears my academic organization has the Team plan and all associated benefits. it still lists a per-seat cost, but I was not advised of any charges. when clicking the button, and do not see anything in the billing section that leads me to believe we will be billed going forward. If we make it to the next month with no charges and the Team plan features intact, I will come back and close this.

**Comment by smasongarrison** [comment]
@samus-aran I am still experiencing this issue. Here is my ticket number. [Ticket 2331182](https://support.github.com/ticket/2331182)

**Comment by clelidm** [comment]
Hi,

I am having the same issue here: https://github.com/DM-Lab-UvA

I recently renewed my teacher account, but lost the team status of the associated organization. 

At [GitHub Global Campus dashboard](https://education.github.com/globalcampus/teacher), the section "Upgrade to GitHub Team" doesn't have the possibility to "Upgrade your academic organizations", but only has the message: 
"You can upgrade your organizations to GitHub Team. Start by creating a new organization."

**Comment by acbart** [comment]
Joining the chorus with my course, it's in the same boat: https://github.com/UD-S24-CISC374

Had my benefits approved, requested to have the organization approved, got the email, but the benefits do not seem to be applied. I cannot request to have the organization's status upgraded. That was about a month ago that I started, so I can't imagine it's just a temporary delay. Is there anything to be done? Or am I just going to have make the students have public repos?

**Comment by jmintser** [comment]
Same here, it's been 3.5 months since our initial ticket.  Help please!
https://support.github.com/ticket/personal/0/2447937
https://support.github.com/ticket/personal/0/2493229

**Comment by ypyou** [comment]
I'm experiencing the same issue. My [ticket](https://support.github.com/ticket/personal/0/2461163) was opened nearly four months ago, yet the problem persists. Can you please assist?
